extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -118,13 +121,82 @@
   transform-origin: right !important;
 }
 
+[info] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: 48px !important; 
+}
+
+[info] [dir=rtl] .i-amphtml-message-container {
+  /* 48px for info button, add width to padding for 
+  * additional story buttons */
+  padding-right: auto !important; 
+  padding-left: 48px !important; 
+}
+
+.i-amphtml-message-container {
+  width: max-content !important;
+  transition-property: opacity, transform !important;
+  float: right !important;
+  /*opacity: 0 !important;*/
+  position: absolute !important;
+  top: 14px !important;
+  right: 48px !important;
+}
+
+[dir=rtl] .i-amphtml-message-container {
+  right: auto !important;
```

Comment:
no need for the `left: 48px` here? I would have guessed you needed it by looking at the rule above

Reply:
oh yeah! thanks! it worked ok before, but the translateX values didn't make sense to me, so this helps a lot!